src/libraries/System.Net.Security/src/System/Net/NegotiateAuthenticationPal.Unix.cs

```diff
@@ -22,7 +22,8 @@ internal partial class NegotiateAuthenticationPal
         private static bool UseManagedNtlm { get; } =
             AppContext.TryGetSwitch("System.Net.Security.UseManagedNtlm", out bool useManagedNtlm) ?
             useManagedNtlm :
-            OperatingSystem.IsMacOS() || OperatingSystem.IsIOS() || OperatingSystem.IsMacCatalyst();
+            OperatingSystem.IsMacOS() || OperatingSystem.IsIOS() || OperatingSystem.IsMacCatalyst() ||
+            (OperatingSystem.IsLinux() && RuntimeInformation.RuntimeIdentifier.StartsWith("linux-bionic", StringComparison.OrdinalIgnoreCase));
```

Comment:
"linux-bionicish" matches this pattern.  Should it?  Since there's no such thing yet, I'd guess not; which means this should use Equals, not StartsWith.

Reply:
> The SDK won't let the "linux-bionicish" (ab)user reach to this point. :)

Currently.  That was really just a standin for "some future RID that has bionic as a prefix".  "bionic2" is where I started, but then decided "bionic2 will probably behave like bionic" was too easy of a defense